spec/models/user_option_spec.rb

```diff
@@ -18,6 +18,28 @@
 
   end
 
+  describe ".mailing_list_mode" do
+    let!(:forum_user) { Fabricate(:user) }
+    let!(:mailing_list_user) { Fabricate(:user) }
+
+    before do
+      forum_user.user_option.update(mailing_list_mode: false)
+      mailing_list_user.user_option.update(mailing_list_mode: true)
+    end
+
+    it "should return false when `SiteSetting.disable_mailing_list_mode` is enabled" do
+      SiteSetting.expects(:disable_mailing_list_mode).twice.returns(true)
```

Comment:
Ah, I was just aping the rest of the tests in the file with the stubbing; seems like instance method test in the file (`.redirected_to_top`) is marked with a '.' as well?